docs/data/date-pickers/base-concepts/ComponentFamilies.js

```diff
@@ -17,9 +16,9 @@ function ProLabel({ children }) {
   return (
     <Stack direction="row" spacing={0.5} component="span">
       <Tooltip title="Included in Pro package">
-        <Link href="/x/introduction/licensing/#pro-plan">
+        <a href="https://mui.com/x/introduction/licensing/#pro-plan">
           <span className="plan-pro" />
```

Comment:
I believe this class is not available in the codesandbox. To fully make the demo work in CSB, could it make sense to make a `styled()` component for this instead? With absolute url to the image?